crypto/merkle/tree.go

```diff
@@ -2,14 +2,16 @@ package merkle
 
 import (
 	"math/bits"
+
+	"github.com/tendermint/tendermint/crypto/tmhash"
 )
 
 // HashFromByteSlices computes a Merkle tree where the leaves are the byte slice,
-// in the provided order.
+// in the provided order. It follows RFC-6962.
 func HashFromByteSlices(items [][]byte) []byte {
 	switch len(items) {
 	case 0:
-		return nil
+		return tmhash.Sum([]byte{})
```

Comment:
It would probably be more consistent with the existing code if there was a function `func emptyHash() []byte` where the other cases are defined too (`leafHash` and `innerHash`)

Reply:
Agreed, done